litmus-portal/frontend/src/components/SideBar/index.tsx

```diff
@@ -182,6 +183,16 @@ const SideBar: React.FC = () => {
         >
           <DocsIcon />
         </CustomisedListItem>
+        <CustomisedListItem
+          key="litmusAPIDocs"
+          handleClick={() => {
+            window.open('https://litmuschaos.github.io/litmus/api.html');
+          }}
```

Comment:
```suggestion
          handleClick={() => window.open('https://litmuschaos.github.io/litmus/api.html')}
```

Reply:
cc: @amityt. Changes are as per discussion. We won't support API docs in the air gap env

Reply:
yes the link is fine you may remove the extra lines due to the { }